myuw/templates/handlebars/card/schedule/visual.html

```diff
@@ -42,15 +42,16 @@ <h3 class="sr-only">Your {{ucfirst quarter}} {{year}} final exam</h3>
 	    </tbody>
 	</table>
     </div>
-
+	<div><p class="sr-only"><a href="#CourseCards">Skip the visual schedule table to Course cards </a></p>
```

Comment:
The phrasing of this is a little awkward. Let's use "Skip the visual schedule and go to course cards".